content/20.treatments.md

```diff
@@ -429,7 +429,9 @@ In mouse models, poly(I:C) and lipopolysaccharide, which are agonists of TLR3 an
 Given the current pandemic, scientists and the medical community are scrambling to repurpose or discover novel host-directed therapies.
 For the general public in particular, whether nutraceuticals or dietary supplements can provide any prophylactic or therapeutic benefit has been a topic of interest.
 Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value [@doi:10.1080/10408390902841529; @doi:10.1038/nrcardio.2016.103].
-The key difference between a dietary supplement and a nutraceutical is that nutraceuticals should not only supplement the diet, but also aid in the prophylaxis and/or treatment of a disorder or disease [@doi:10.1208/ps050325].
+The key difference between a dietary supplement and a nutraceutical is that nutraceuticals should not only supplement the diet, but also aid in the prophylaxis and/or treatment of a disorder or disease [@doi:10.1208/ps050325]. 
+In the United States, dietary supplement and nutraceutical sales were worth $345 million for the entirety of 2019, whereas sales reached $435 million during the six-week period of the outbreak that ended on April 5 of 2020, which grew by a further $151 million by May 17 2020 [@url:https://www.nutritionaloutlook.com/view/peak-dietary-supplement-sales-leveling-during-covid-19-pandemic-growth-still-remains-strong].
+The increase in sales was driven by a consumer perception that dietary supplements and nutraceuticals would protect consumers from infection and/or mitigate the impact of infection due to the various 'immune-boosting' claims of these various products [@doi:10.1016/j.tifs.2020.09.001; @url:https://kerry.com/insights/kerrydigest/2020/5-food-and-beverage-trends-in-europe-during-covid-19].
```

Comment:
```suggestion
The increase in sales was driven by a consumer perception that dietary supplements and nutraceuticals would protect consumers from infection and/or mitigate the impact of infection due to the various 'immune-boosting' claims of these various products [@doi:10.1016/j.tifs.2020.09.001; @url:https://kerry.com/insights/kerrydigest/2020/5-food-and-beverage-trends-in-europe-during-covid-19].
Therefore, these alternative treatments and prophylactics are also important to consider in the context of COVID-19, especially since their regulation is less rigorous than that of pharmaceuticals in many places.
```
Just an idea to add one more sentence really bringing the message home!

Reply:
Great idea, and well put. Thanks a million.